Motorcycle.h

```diff
@@ -1,15 +1,17 @@
 #pragma once
-#include "Vehicle.h"
 
 #include <iostream>
 #include <limits>
+#include "Vehicle.h"
 
-struct Motorcycle : public Vehicle
+struct Motorcycle : Vehicle
 {
     Motorcycle(const std::string& n);
 
-
-    
+    ~Motorcycle() override = default;
+    Motorcycle(const Motorcycle&) = default;
+    Motorcycle& operator=(const Motorcycle&) = default;
+   
```

Comment:
These should be defined outside of the class.
- [ ] revise

Reply:
The motorcycle also has the tryToEvade() function that Car has so I just removed these like I did with Car...